src/sidebar.js

```diff
@@ -175,7 +183,7 @@ module.exports = {
     },
     {
       type: "category",
-      label: "OAuth2 & OpenID",
+      label: "OAuth2 and OIDC",
```

Comment:
The product is called: Ory OAuth2 and OpenID, see: https://docs.google.com/document/d/1GWNnI4dyVlXZzKMLPMPiuvz3rQJHf_P_jg3flsUq-ec/edit

Reply:
OpenID is a recognized name, experts know what OIDC means but many do not. Let's please stick to the product name here. We can use the 100% accurate terminology in the documentation:

```suggestion
      label: "OAuth2 and OpenID",
```